CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
CMake 3.18 is required for CMAKE_CUDA_STANDARD 17

Reply:
Thanks!